lib/perl/Genome/InstrumentData/AlignmentResult/Merged.pm

```diff
@@ -191,7 +196,9 @@ sub create {
         } else {
             #just collect the BAMs for a merge
             for my $alignment (@alignments) {
-                push @bams_for_final_merge, $alignment->alignment_bam_file_paths;
+                my $temp_allocation = $self->_get_temp_allocation($alignment);
```

Comment:
This line is duplicated from line 178  - should it be moved outside the if/else block?